PyStemmusScope/global_data/assets/lccs_to_igbp_table.csv

```diff
@@ -0,0 +1,24 @@
+"lccs_class","IGBP_STEMMUS_SCOPE"
```

Comment:
We should explain how this file is generated. I suggest adding a section "Dataset" to the documentation and explaining all datasets there. This can be done in another item. see issue #78